client/sites-dashboard/components/sites-table-row/docs/example.jsx

```diff
@@ -6,145 +7,13 @@ const CONTAINER_STYLES = {
 	padding: '16px',
 };
 
-const exampleSite = {
-	URL: 'https://wpvip.com',
-	is_coming_soon: false,
-	is_private: false,
```

Comment:
Well done moving the examples to a different file and reusing them in both examples 👌